shell/platform/common/accessibility_bridge_unittests.cc

```diff
@@ -37,9 +37,8 @@ FlutterSemanticsNode CreateSemanticsNode(
 }
 
 TEST(AccessibilityBridgeTest, basicTest) {
-  std::shared_ptr<AccessibilityBridge> bridge =
-      std::make_shared<AccessibilityBridge>(
-          std::make_unique<TestAccessibilityBridgeDelegate>());
+  std::shared_ptr<TestAccessibilityBridge> bridge =
+      std::make_shared<TestAccessibilityBridge>();
```

Comment:
Just curious, do you know why these are `std::shared_ptr`s? It looks like they could be `std::unique_ptr`s instead

Reply:
Done.